lib/logflare/backends/adaptor/postgres_adaptor/shared_repo.ex

```diff
@@ -44,6 +44,7 @@ defmodule Logflare.Backends.Adaptor.PostgresAdaptor.SharedRepo do
 
         opts ++ fields
       end
+    opts = opts ++ (Map.take(config, [:socket_options]) |> Map.to_list())
```

Comment:
```suggestion
    opts = opts ++ [socket_options: config[:socket_options]]
```

Reply:
The advantage of `take` would be that it also works if the `:socket_options` are omitted, right?